crates/epaint/src/tessellator.rs

```diff
@@ -864,19 +875,58 @@ fn stroke_path(
     feathering: f32,
     path: &[PathPoint],
     path_type: PathType,
-    stroke: Stroke,
+    stroke: &PathStroke,
     out: &mut Mesh,
 ) {
     let n = path.len() as u32;
 
-    if stroke.width <= 0.0 || stroke.color == Color32::TRANSPARENT || n < 2 {
+    if stroke.width <= 0.0 || stroke.color == ColorMode::Solid(Color32::TRANSPARENT) || n < 2 {
```

Comment:
we can add a helper for this

```suggestion
    if stroke.width <= 0.0 || stroke.color == ColorMode::TRANSPARENT || n < 2 {
```

Reply:
resolved with 2d8ab0ce5d44c936657d61f815ca3eb4426febe4